### PR TITLE
docs(ivy): add size debugging section

### PR DESCRIPTION
**Pull request by kara**

### CODE REVIEW

File: aio/content/guide/ivy-compatibility.md [hunk 1]
```diff
@@ -17,6 +17,20 @@ If you're still seeing the errors, they are not specific to Ivy. In this case, y
 
 If the errors are gone, switch back to Ivy by removing the changes to the `tsconfig.json` and review the list of expected changes below.
 
+{@a size-debugging}
+### Size Debugging
```
Comment (IgorMinar) [inline aio/content/guide/ivy-compatibility.md]:
```suggestion
### Payload size debugging
```

File: aio/content/guide/ivy-compatibility.md [hunk 2]
```diff
@@ -17,6 +17,20 @@ If you're still seeing the errors, they are not specific to Ivy. In this case, y
 
 If the errors are gone, switch back to Ivy by removing the changes to the `tsconfig.json` and review the list of expected changes below.
 
+{@a size-debugging}
```
Comment (IgorMinar) [inline aio/content/guide/ivy-compatibility.md]:
```suggestion
{@a payload-size-debugging}
```

**Review by IgorMinar — APPROVED** [review]
Lgtm otherwise. Thanks

File: aio/content/guide/ivy-compatibility.md [hunk 3]
```diff
@@ -17,6 +17,20 @@ If you're still seeing the errors, they are not specific to Ivy. In this case, y
...
+### Size Debugging
+
+If you notice that the size of your application's main bundle has increased with Ivy, you may want to check the following:
+
+1. Verify that the components and `NgModules` that you want to be lazy loaded are only imported in lazy modules.
+Anything that you import outside lazy modules can end up in the main bundle. 
+See more details in the original issue [here](https://github.com/angular/angular-cli/issues/16146#issuecomment-557559287). 
+
+1. Check that imported libraries have been marked side-effect-free.
```
Comment (BO41) [inline aio/content/guide/ivy-compatibility.md]:
```suggestion
2. Check that imported libraries have been marked side-effect-free.
```

Comment (kara) [reply aio/content/guide/ivy-compatibility.md]: With markdown, you actually want all "1s" to create ordered lists. It renders in numeric order, so if you switch items around, you don't have to update the numering.

**Pull request closed without merging**